base/number.jl

```diff
@@ -136,6 +136,52 @@ true
 """
 signbit(x::Real) = x < 0
 
+"""
+    ispositive(x)
+
+Test whether `x > 0`. See also [`isnegative`](@ref).
+
+!!! compat "Julia 1.11"
+    This function requires at least Julia 1.11.
```

Comment:
```suggestion
!!! compat "Julia 1.12"
    This function requires at least Julia 1.12.
```
The feature freeze for 1.11 was last month and it's already in alpha. I expect this to hit in 1.12.